.github/workflows/compile.yml

```diff
@@ -0,0 +1,30 @@
+name: Compile source code
+
+on:
+  pull_request:
+    branches:
```

Comment:
Please remove the branches restriction, as a lot of contributors use their own branch

Reply:
Done